emote.py

```diff
@@ -46,9 +57,12 @@ def emoteadd(self, emote):
 
 		return 'Emote "' + args[0] + '" added :)'
 
+	def emotelist(self):
+		return 'Emotes: ' + ', '.join(list(self.emotes.keys()))
+
 	def emotedel(self, emote):
 		if emote == 'lenny':
-			return 'One cannot simply delete Lenny ( ͡° ͜ʖ ͡°)'
+			return 'One cannot simply delete Lenny ' + self.emotes['lenny'][0]
```

Comment:
Totally unimportant: this could be a call to `self.emote`, which would rotate the lenny used here, in case more lennys were added for some reason. Otherwise, this will always be the same lenny.